src/warp.js

```diff
@@ -19,6 +19,7 @@ eventEmitter.on('progress-notification', (data) => {
 });
 
 const warp = WarpFactory.forMainnet()
+  .useGwUrl(config.gwUrl)
```

Comment:
please add default value to .env.default

Reply:
ok